orchestrator.py

```diff
@@ -33,7 +35,7 @@ def __init__(self, broker_ip, port=1883):
 
     def publish_camera_status(self, message: str) -> None:
         """ Send a message on the current device's camera status topic. """
-        status_topic = f"{topics.DAShboard.status_camera}/{self.device}"
+        status_topic = str(topics.Camera.status_camera / self.device)
```

Comment:
Why the change from f-strings to `str()`?

Reply:
We are using the `Topic.__str__()` method for conversion to strings to make use of `Topic.__trudiv__()`.

Reply:
What Nick said! Basically we're abstracting away the string building process altogether, but still need to cast to a string at the end. Much the same thing, but we get to flex some cool syntax. (If you didn't notice - that `/` isn't part of a string, it's the operator `/` that has been overloaded in the `Topic` class. Take a look!)